ruby.rb

```diff
@@ -5,46 +5,77 @@
 tempArray = [1, 2, 6, 9, 3, 21]
 
 # your ruby loop here
+puts tempArray.map{|x| x * 2}
 
+tempArray.each do |x|
+   puts x * 2
+end
 
 #### 2. From all the built in Ruby methods we've seen in class this week, choose three that you think are particularly helpful and create examples to show how they work.
 
 # your three built in ruby methods
 
+# The delete method to remove unwanted characters
+string = 'hello'
+puts string.delete 'h'
+
+# The reverse method
+arrayEx = [2,3,4,5,6,7]
+puts arrayEx.reverse
+
+# The to string method to convert a data type to a string
+puts arrayEx.to_s
 
 #### 3. Create a method that takes in a sentence and returns a new sentence with the first letter of each word capitalized.
 
 sentence = "hello there, how are you?"
 expected output = "Hello There, How Are You?"
 
+def firstLetter(string)
+ puts string.split.map{|x| x.capitalize}.join(' ')
+end
+
+firstLetter(sentence)
+
 #### 4.  Create a method that takes in a string and returns a new string with all the vowels removed. HINT:  there's a built in string method for this
 
 no_vowels = "I have no vowels"
 expected output = " hv n vwls"
 
+puts no_vowels.delete('aeiouAEIOU')
 
 #### 5. Look at this horrible ruby code, and fix it to be good ruby code.
 
```

Comment:
You want to make sure you are evaluating against your instance variable here. Also ruby doesn't need the () in the if/else blocks.